rdt/transformers/categorical.py

```diff
@@ -516,6 +527,7 @@ def _reverse_transform(self, data):
         Returns:
             pandas.Series
         """
+        check_nan_in_transform(data, self._is_integer)
```

Comment:
Assign it to something.

Reply:
The FrequencyEncoder doesn't need the output of `check_nan_in_transform` because it converts to float somewhere else. The function is just call to raise the warning if we are in the situation of the issue